content/docs/distributed-asynchronous.md

```diff
@@ -0,0 +1,51 @@
+# Message delivery patterns in asynchronous systems
+
+First of all we should ask ourselves a question: why both with distributed asynchronous systems? Wouldn't it be better to just build synchronous non-distributed ones? It certainly would be easier.
+
+### Why distributed
+
+A system is *distributed* if it consists of multiple components and these components are running inside different processes. The fact that these components are part of a single system means they need to communicate to realise functions of that systems. If there was no requirement for communication then we could call them separate non-distributed systems.
```

Comment:
I would add one more thing to this: Distributed system consists of components that can fail independently. In non-distributed systems, it's either on or off. Distributed one has all shades of grey.